docs/source/usersguide/processing.rst

```diff
@@ -6,31 +6,33 @@ Data Processing and Visualization
 
 This section is intended to explain in detail the recommended procedures for
 carrying out common post-processing tasks with OpenMC. While several utilities
-of varying complexity are provided to help automate the process, in many cases
-it will be extremely beneficial to do some coding in Python to quickly obtain
-results. In these cases, and for many of the provided utilities, it is necessary
-for your Python installation to contain:
-
-* [1]_ `Numpy <http://www.numpy.org/>`_
-* [1]_ `Scipy <http://www.scipy.org/>`_
-* [2]_ `h5py <http://code.google.com/p/h5py/>`_
-* [3]_ `Matplotlib <http://matplotlib.org/>`_
+of varying complexity are provided to help automate the process, the most
+powerful capabilities for post-processing derive from use of the :ref:`Python
+API <pythonapi>`. Both the provided scripts and the Python API rely on a number
+third-party Python packages, including:
+
+* [1]_ `NumPy <http://www.numpy.org/>`_
+* [2]_ `h5py <http://www.h5py.org>`_
+* [3]_ `pandas <http://pandas.pydata.org>`_
+* [3]_ `matplotlib <http://matplotlib.org/>`_
 * [3]_ `Silomesh <https://github.com/nhorelik/silomesh>`_
 * [3]_ `VTK <http://www.vtk.org/>`_
+* [3]_ `lxml <http://lxml.de>`_
 
-Most of these are easily obtainable in Ubuntu through the package manager, or
-are easily installed with distutils.
+Most of these are can easily be installed with `pip <https://pip.pypa.io>`_
+or alternatively obtaining through a package manager.
 
-.. [1] Required for tally data extraction from statepoints with statepoint.py
-.. [2] Required only if reading HDF5 statepoint files.
-.. [3] Optional for plotting utilities
+.. [1] Required for most post-processing tasks
+.. [2] Required for reading HDF5 output files
+.. [3] Not used directly by the Python API, but are optional dependencies for a
```

Comment:
I would say that "pandas" is used directly by the Python API, but is "optional" - minor clarification, you can decide whether or not it is necessary.